python/interpret_community/explanation/explanation.py

```diff
@@ -1866,7 +1866,7 @@ def _get_value_from_file(file_var):
     if meta == 'DataFrame':
         return pd.DataFrame(data)
     elif meta == 'DatasetWrapper':
-        return DatasetWrapper(data)
+        return DatasetWrapper(np.array(data))
```

Comment:
Hmm, I'm not sure if this is fully correct.  The data here could be scipy sparse, numpy dense or a pandas DataFrame - all of these could be input into a DatasetWrapper.  On the other hand I know this save_explanation code doesn't support sparse data currently at all, there is a work item to add this support:
https://github.com/interpretml/interpret-community/issues/378
this code is pretty flimsy honestly.
I think in ideal case we should actually be passing the pandas or numpy array to the DatasetWrapper for full correctness (and scipy sparse once we get to supporting it here).  But maybe to unblock this issue currently this is fine as is.

Reply:
Maybe I could add a check to convert into numpy array if it is an instance of a list? 

I think DatasetWrapper is missing some validation code to cause the supported types. I guess we could also add a check in DatasetWrapper to make the behavior more deterministic and fail in case unsupported types are injected into that class.

Reply:
Yeah, I think this save/load logic isn't very robust and needs more changes/testing.  But it's ok to merge this as is for now to unblock the current issue.